cli/packages/prisma-cli/package.json

```diff
@@ -48,13 +50,14 @@
   "scripts": {
     "build": "rimraf dist && tsc -d && chmod +x dist/index.js",
     "lint": "tslint src/**/*.ts",
-    "publish-homebrew": "dotenv -e scripts/.env ts-node ./scripts/release-brew.ts",
-    "make-binary": "npm run build && npm run package",
+    "publish-binaries": "dotenv -e scripts/.env ts-node ./scripts/release-binaries.ts",
+    "binary-osx": "npm run build && npm run package",
+    "binary-linux": "npm run build && npm run package-linux",
```

Comment:
Either keep one of the two `binary-osx`, `package` and `binary-linux` as they do the same thing OR move them together in `package.json` as that is more readable.